tests/ha/hawk_gui.pm

```diff
@@ -46,7 +46,8 @@ sub run {
     install_docker;
 
     # TODO: Use another namespace using team group name
```

Comment:
I've been meaning to add a comment here with information regarding where the source code to the python test and docker image is hosted. Can you add something like:
```
# Docker image source in https://github.com/ricardobranco777/hawk_test.
# It will be eventually moved to https://github.com/ClusterLabs/hawk/e2e_test
```
Thanks!

Reply:
Sure!